data/help.html

```diff
@@ -1,57 +1,42 @@
 <h2>Help</h2>
 
-<p>New here? <strong>geojson.io</strong> is a quick, simple tool for creating,
-viewing, and sharing maps. geojson.io is named after <a href='http://geojson.org/' target='_blank'>GeoJSON</a>,
-an open source data format, and it supports GeoJSON in all ways - but also
-accepts KML, GPX, CSV, GTFS, TopoJSON, and other formats.</p>
+<p><strong>geojson.io</strong> is a quick, simple tool for creating,
+    viewing, and sharing maps. geojson.io is named after <a href='http://geojson.org/' target='_blank'>GeoJSON</a>,
+    an open source spatial data format, and it supports GeoJSON in all ways - but also
+    accepts KML, GPX, CSV, GTFS, TopoJSON, and other formats.</p>
 
-<p>Need extra help or see a bug? <a target='_blank' href='https://github.com/mapbox/geojson.io/issues?state=open'>Open an issue
-    on geojson.io's issue tracker.</a>
+<p>Want to request a feature or report a bug? <a target='_blank'
+        href='https://github.com/mapbox/geojson.io/issues?state=open'>Open
+        an issue
+        on geojson.io's issue tracker.</a>
```

Comment:
As an "open" project, I think this is the right call, but we might consider mirroring these issues to Jira for our own sanity/compliance.

Reply:
I think we can leave it as is until it becomes a problem.  I'll be cleaning up the issue queue as part of this launch.